packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -739,8 +739,8 @@ export class PaginatedMessage {
 		for (const [idx, embed] of Object.entries(data.embeds)) {
 			if (embed) {
 				embed.footer ??= { text: this.template.embeds?.[Number(idx)]?.footer?.text ?? this.template.embeds?.[0]?.footer?.text ?? '' };
-				embed.footer.text = `${this.pageIndexPrefix ? `${this.pageIndexPrefix} ` : ''}${index + 1} / ${this.pages.length}${
-					embed.footer.text ? ` • ${embed.footer.text}` : ''
+				embed.footer.text = `${this.pageIndexPrefix ? `${this.pageIndexPrefix} ` : ''}${index + 1} / ${this.pages.length} | ${
```

Comment:
The idea is that the `|` is only appended when there's a footer text.
```suggestion
				embed.footer.text = `${this.pageIndexPrefix ? `${this.pageIndexPrefix} ` : ''}${index + 1} / ${this.pages.length}${
```